inlang/source-code/cli/src/utilities/getInlangProject.ts

```diff
@@ -10,17 +11,43 @@ export async function getInlangProject(args: { projectPath: string }): Promise<I
 	const baseDirectory = process.cwd()
 	const projectPath = resolve(baseDirectory, args.projectPath)
 
-	const project = await loadProject({
-		projectPath,
-		nodeishFs: fs,
-		_capture(id, props) {
-			telemetry.capture({
-				// @ts-ignore the event types
-				event: id,
-				properties: props,
-			})
-		},
-	})
+	const repoRoot = await findRepoRoot({ nodeishFs: fs, path: projectPath })
+
+	let project
+	if (!repoRoot) {
+		console.error(
+			`Could not find repository root for path ${projectPath}, falling back to direct fs access`
+		)
+
+		project = await loadProject({
+			projectPath,
+			nodeishFs: fs,
+			_capture(id, props) {
+				telemetry.capture({
+					// @ts-ignore the event types
+					event: id,
+					properties: props,
+				})
+			},
+		})
+	} else {
```

Comment:
I vote for just throwing. 

We will complicate our code and architecture if we bake conditional logic everywhere, whether a repo exists or not. Inlang requires lix and lix requires repositories.

Reply:
What do you think?

Reply:
The throw can happen in `openRepository` a la "no repository path provided. If you are using `findRoot` no root has been found."

Reply:
@samuelstroschein  i could throw in the cli and show the error: you need to use the cli inside a git repository, effectively removing support for other version control systems and also using the cli without having a repo. i figured we better do that step when we remove the deprecated nodishFS argument.